com/haxepunk/Cursor.hx

```diff
@@ -0,0 +1,29 @@
+package com.haxepunk;
+
+import com.haxepunk.Entity;
+import com.haxepunk.graphics.Graphiclist;
+import openfl.ui.Mouse;
+
+class Cursor extends Entity
+{
+	/**
+	 * Constructor.
+	 * @param	graphic		Graphic to assign to the Entity.
+	 * @param	mask		Mask to assign to the Entity.
+	 */
+	public function new(graphic:Graphic = null, mask:Mask = null)
```

Comment:
I made sure I added with this in mind: http://www.kongregate.com/games/coolio_niato/streamline

I think having the option to have the cursor also be a "player" is important.